src/screens/WaitTimeScreen.tsx

```diff
@@ -271,7 +272,7 @@ export const WaitTimeScreen = ({ navigation }: IWaitTimeScreenProps) => {
             <RefreshControl
               refreshing={refreshing}
               onRefresh={onRefresh}
-              progressBackgroundColor={theme.iconColor}
+              progressBackgroundColor={theme.refreshIconBackgroundColorAndroid} // Android only: Background color of refresh icon
```

Comment:
Doesn't `colours` prop work?

Reply:
Sorry what are you suggesting?

Reply:
Im not sure what this does, but this https://reactnative.dev/docs/refreshcontrol#colors-android for android